content/docs/2.14/operate/events.md

```diff
@@ -46,9 +46,15 @@ metadata:
   name: {cloud-event-name}
 spec:
   clusterName: {cluster-name} #Optional. Will be used in the source/subject to specify where the event comes from. The default value is 'kubernetes-default' and it can also be set during the installation of KEDA with --k8sClusterName. This one will overwrite others if set.
+  authenticationRef: 
+    name: {trigger-authentication-name} #Optional. Used to reference a `TriggerAuthentication` for authentication. 
+    kind: ClusterTriggerAuthentication #Optional. Used to choose the authenticatio scopes. https://keda.sh/docs/2.13/concepts/authentication/#authentication-scopes-namespace-vs-cluster
```

Comment:
Let's use namespace-scoped TA by default
```suggestion
    name: {trigger-authentication-name} #Optional. Used to reference a `TriggerAuthentication` for authentication. 
    kind: TriggerAuthentication # Optional. Used to choose the authentication scopes. https://keda.sh/docs/latest/concepts/authentication/#authentication-scopes-namespace-vs-cluster
```

Is `name` really optional? I presume it is only when `azureEventGridTopic` is not used?

Reply:
It's optional. I reused the ta logic of ScaledObejct. So all properties requirement are the same as ScaledObejct.